x/il/types/params.go

```diff
@@ -0,0 +1,57 @@
+package types
+
+import (
+	"errors"
+	"fmt"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// DefaultParamspace defines default space for oracle params
+const DefaultParamspace = ModuleName
+
+// Parameter keys
+var (
+	ParamStoreKeyContractAddress = []byte("ContractAddress")
```

Comment:
cc @zmanian do we want to store this here? This would be the pattern moving forward for new modules (i.e. module defines it's strategy contract)